spec/system/provider_interface/make_offer_js_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+
+RSpec.feature 'Provider makes an offer', type: :system, js: true do
+  include DfESignInHelpers
+  include ProviderUserPermissionsHelper
+
+  let(:provider_user) { create(:provider_user, :with_dfe_sign_in) }
+  let(:provider) { provider_user.providers.first }
+  let(:ratifying_provider) { create(:provider) }
+
+  let(:application_form) { build(:application_form, :minimum_info) }
+  let(:course) { build(:course, :open_on_apply, provider: provider, accredited_provider: ratifying_provider) }
+  let(:course_option) { build(:course_option, course: course) }
+
+  scenario 'Setting offer conditions' do
```

Comment:
None of the behaviour I've implemented is any different from the non-JS behaviour, so it's hard to prove that JS is running and working.

It might be nice to have this same spec run in JS and non-JS modes